opentrons-ai-client/src/resources/types.ts

```diff
@@ -85,6 +85,15 @@ export interface Chat {
   attachments?: FileAttachment[]
 }
 
+/** Chat message serialized for API requests (without File content) */
+export interface ChatMessage {
+  role: Role
+  content: string
+  protocol_content?: string
+  protocol_format?: ProtocolFormat
```

Comment:
typescript likes camelCase.

```suggestion
  protocolContent?: string
  protocolFormat?: ProtocolFormat
```

Reply:
@koji  this is something we talked with Shlok, we followed this convention because backend is `snake_case`. 
Let's open another PR to fix this. It would be great if you suggest me how to best handle such cases that frontend wants camelCase and backend wants snake_case.